File: C++/BroCode/HypotenuseCalculator.cpp
#include <iostream>
#include <cmath>

int main() 
{
   double a;
   double b;
   double c;

   std::cout << "Enter side A: ";
   std::cin >> a;

   std::cout << "Enter size B: ";
   std::cin >> b;

   c = sqrt(pow(a, 2) + pow(b, 2));

   std::cout << "side C: " << c;

   return 0;
}

File: C#/BroCode/HypotenuseCalculator.cs
using System;

namespace HypotenuseCalculator
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Enter Side A: ");
            double a = Convert.ToDouble(Console.ReadLine());

            Console.WriteLine("Enter Side B: ");
            double b = Convert.ToDouble(Console.ReadLine());

            double c = Math.Sqrt((a*a)+(b*b));

            Console.WriteLine("The Hypotenuse is: " + c);

            Console.ReadKey();
        }
    }
}
